Cython/Compiler/Symtab.py

```diff
@@ -1362,6 +1364,31 @@ def declare_tuple_type(self, pos, components):
         ttype.entry = entry
         return entry
 
+    def declare_defaults_c_class(self, pos, components):
+        # returns an entry (for the c-class)
+        components = tuple(components)
+        try:
+            entry = self._cached_defaults_c_class_entries[components]
+        except KeyError:
+            cname = self.next_id(Naming.defaults_struct_prefix)
+            cname = EncodedString(cname)
+            entry = self._cached_defaults_c_class_entries[components] = self.declare_c_class(
+                cname, pos, defining=True, implementing=True,
+                objstruct_cname=cname)
+            self.check_c_class(entry)
+            entry.type.is_final_type = True
+            scope = entry.type.scope
+            scope.is_internal = True
+            scope.is_defuults_class_scope = True
+
+            # zero pad the argument number so they can be sorted
+            num_zeros = math.floor(math.log10(len(components)))
+            format_str = "arg{0:0%dd}" % num_zeros
+            for n, type_ in enumerate(components):
+                scope.declare_var(EncodedString(format_str.format(n)), type_, None,
+                                  is_cdef = True)
+        return entry
+
```

Comment:
Let's avoid the lengthy indented block when we already know we're done.
```suggestion
    def declare_defaults_c_class(self, pos, components):
        # returns an entry (for the c-class)
        components = tuple(components)
        try:
            return self._cached_defaults_c_class_entries[components]
        except KeyError:
            pass

        cname = self.next_id(Naming.defaults_struct_prefix)
        cname = EncodedString(cname)
        entry = self._cached_defaults_c_class_entries[components] = self.declare_c_class(
            cname, pos, defining=True, implementing=True,
            objstruct_cname=cname)
        self.check_c_class(entry)
        entry.type.is_final_type = True
        scope = entry.type.scope
        scope.is_internal = True
        scope.is_defuults_class_scope = True

        # zero pad the argument number so they can be sorted
        num_zeros = math.floor(math.log10(len(components)))
        format_str = "arg{0:0%dd}" % num_zeros
        for n, type_ in enumerate(components):
            scope.declare_var(EncodedString(format_str.format(n)), type_, None, is_cdef = True)
        return entry

```